packages/@react-spectrum/progress/stories/ProgressBar.stories.tsx

```diff
@@ -37,11 +29,22 @@ const grayedBoxStyle: CSSProperties = {
 };
 
 storiesOf('Progress/ProgressBar', module)
-  .addParameters({providerSwitcher: {status: 'positive'}})
-  .addDecorator(withKnobs)
+  .addParameters({
+    providerSwitcher: {status: 'positive'},
+    args: {value: 32},
+    argTypes: {
+      value: {
```

Comment:
Is this just an example? Shouldn't we wire up `showValueLabel`, `size`, `labelPosition`, etc.?

Reply:
I ported the existing knobs to controls for this PR. We can certainly add more later.